src/kafka_consumer.cpp

```diff
@@ -70,6 +70,17 @@
 #include "bsmfilter.hpp"
 #include "cvlib.hpp"
 
+using namespace std;
+
+const char* getEnvironmentVariable(const char* variableName) {
+    const char* toReturn = getenv(variableName);
+    if (!toReturn) {
+        cout << "[ERROR] Something went wrong attempting to retrieve the environment variable " << variableName << endl;
```

Comment:
Is it possible to swap this to the logger rather than cout?

Reply:
There is an internal and an external logger specified in the PPM class, so they can only be used in methods within that class. If we wanted to use the logger here we'd probably have to do some refactoring and create a shared logger class that can be utilized by the entire project.

Reply:
Should I create a work item for this?

Reply:
Let's go ahead and add this to the backlog.